### ISSUE TITLE
Remember/Sticky/Save timers across restarts

### ISSUE THREAD

**Issue by Wisdawn** [Dziemborowicz/Hourglass#262]
I just noticed that after restarting Windows, I lose all my timers and their titles. This is almost a deal-breaker for me.

I would really, really appreciate the option to remember/stick/save timers, or make certain timers permanent, so I don't have to name and organize them every time I restart my operating system.

Thank you.

**Comment by i2van** [comment]
@Wisdawn Fixed in https://github.com/i2van/hourglass

Also check:

![image](https://github.com/user-attachments/assets/119b8e9a-7018-436b-a9f6-045d412d13a6)

**Comment by Wisdawn** [comment]
![20241105_2104_165](https://github.com/user-attachments/assets/980a064d-ce9a-4ea7-863c-68bc1708eb86)

Thank you for bringing the feature to my attention, @i2van! I have now ticked the option to "Open saved timers on startup," but _how_ do I save any timer? I don't see the option to save anywhere, and as you can see in the screenshot attached, Hourglass says I have "No saved timers." I confirm that I have set up three timers, though.

**Comment by i2van** [comment]
@Wisdawn You don't need to save timers **explicitly**, timer is saved when you close not yet expired one (don't ask me why it was implemented this way). Just close application and run it again without any command-line and all timers you have should be automatically loaded (if the corresponding option is set).

By the way, please use the [Hourglass modified by me](https://github.com/i2van/hourglass), old one has a lot of bugs I've fixed already.

**Comment by Wisdawn** [comment]
@i2van Yes, I just realized that the timers save in many ways, including stopping and then restarting the timer! I restarted my system the other day to see dozens of duplicate timers when I started Hourglass. This is bizarre design.

I just installed your version. It does work much better, but it still lacks a Save button/link and preventing the saving of timers any other way. 

Timers shouldn't save by stopping, starting, or in any way. There should simply be a Save button or link for saving, and that's it.

Thank you very much for working on your version. I was wondering why this useful app has been abandoned for so long and why there wasn't someone else working on updating it.

**Comment by i2van** [comment]
Tracking: https://github.com/i2van/hourglass/issues/24

**Comment by i2van** [comment]
Fixed in [1.15.45](https://github.com/i2van/hourglass/releases/tag/1.15.45)
